docs/user/cli-reference.md

```
@@ -1361,14 +1363,14 @@ When generating bundles with nodewright-customizations enabled, validation warni
 1. **Workload Selector Warning**: When nodewright-customizations is enabled with training intent, if `--workload-selector` is not set, a warning will be displayed:
 
 ```
-Warning: nodewright-customizations is enabled but --workload-selector is not set. 
+Warning: nodewright-customizations is enabled but --workload-selector is not set.
 This may cause nodewright to evict running training jobs. Consider setting --workload-selector to prevent eviction.
 ```
 
 2. **Accelerated Selector Warning**: When nodewright-customizations is enabled with training or inference intent, if `--accelerated-node-selector` is not set, a warning will be displayed:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)